libs/constrained-generators/src/Constrained/Spec/Map.hs

```diff
@@ -52,21 +62,33 @@ data MapSpec fn k v = MapSpec
   , mapSpecSize :: Specification fn Integer
   , mapSpecElem :: Specification fn (k, v)
   , mapSpecFold :: FoldSpec fn v
+  , mapSpecMinValSize :: Maybe (MinValSize fn v)
+  -- Specifaction of the number of unique Values in the range of the Map that meet the Mapspec.
+  -- The Maybe is there because Nothing means (MinValSpec TrueSpec),
+  -- but does require the (Ord v) constraint
```

Comment:
does not require the Ord v constraint?

Reply:
The (Ord v) constraint is carried by the MinValSize constructor, So it is satisfied locally, only when it is used.  If that was not the case, every MapSpec would need an (Ord v) constraint.
If this isn't clear we shpould talk about it.